src/univ/menu.cpp

```diff
@@ -2314,6 +2332,11 @@ void wxMenuBar::OnKeyDown(wxKeyEvent& event)
     }
 }
 
+void wxMenuBar::OnCaptureLost(wxMouseCaptureLostEvent& event)
```

Comment:
```suggestion
void wxMenuBar::OnCaptureLost(wxMouseCaptureLostEvent& WXUNUSED(event))
```
to fix the warning in wxX11 build.